drivers/bridge/bridge.go

```diff
@@ -483,7 +484,7 @@ func parseNetworkOptions(id string, option options.Generic) (*networkConfigurati
 
 	if val, ok := option[netlabel.Internal]; ok {
 		if internal, ok := val.(bool); ok && internal {
-			return nil, &driverapi.ErrNotImplemented{}
+			config.Internal = true
 		}
 	}
 
```

Comment:
In this file we also need to skip the call to `func (n *bridgeNetwork) isolateNetwork()` if "config.Internal == true"